config/locales/metadata/en.yml

```diff
@@ -496,7 +496,7 @@ en:
           ebi_db: ENA-STUDY
 
         hmdmc_approval_number:
```

Comment:
Is it worth changing this key to `humfre_approval_number` too?

Reply:
We want to keep it as hmdmc_approval_number internally in fields and in DB columns. We change the label texts only.